webrtc-stats.html

```diff
@@ -1795,31 +1756,42 @@ <h2>
                 </p>
               </dd>
               <dt>
-                <dfn><code>estimatedPlayoutTimestamp</code></dfn> of type <span class=
-                "idlMemberType"><a>DOMHighResTimeStamp</a></span>
+                <dfn><code>priority</code></dfn> of type <span class="idlMemberType"><a>
+                RTCPriorityType</a></span>
               </dt>
               <dd>
                 <p>
-                  Only valid for remote sources. This is the estimated playout time of this track.
-                  The playout time is the NTP timestamp of the last playable audio sample or video
-                  frame that has a known timestamp (from an RTCP SR packet mapping RTP timestamps
-                  to NTP timestamps), extrapolated with the time elapsed since it was ready to be
-                  played out. This is the "current time" of the track in NTP clock time of the
-                  sender and can be present even if there is no audio or video currently playing.
-                </p>
-                <p>
-                  This can be useful for estimating how much audio and video is out of sync for two
-                  tracks from the same source, <code>audioTrackStats.estimatedPlayoutTimestamp -
-                  videoTrackStats.estimatedPlayoutTimestamp</code>.
+                  Indicates the priority set for the track. It is specified in
+                  [[!RTCWEB-TRANSPORT]], Section 4.
                 </p>
               </dd>
+            </dl>
+          </section>
+        </div>
+      </section>
+      <section id="vststats-dict*">
+        <h3>
+          <dfn>RTCVideoStreamTrackStats</dfn> dictionary
+        </h3>
+        <div>
+          <pre class="idl">dictionary RTCVideoStreamTrackStats : RTCMediaStreamTrackStats {
+             unsigned long       frameWidth;
+             unsigned long       frameHeight;
+             double              framesPerSecond;
+};</pre>
+          <section>
+            <h2>
+              Dictionary <a class="idlType">RTCVideoStreamTrackStats</a> Members
+            </h2>
+            <dl data-link-for="RTCVideoStreamTrackStats" data-dfn-for="RTCVideoStreamTrackStats"
+            class="dictionary-members">
               <dt>
                 <dfn><code>frameWidth</code></dfn> of type <span class="idlMemberType"><a>unsigned
                 long</a></span>
               </dt>
               <dd>
                 <p>
-                  Only valid for video MediaStreamTracks and represents the width of the last
+                  Represents the width of the last
                   processed video frame for this track. Before the first frame is processed this
```

Comment:
nit: There are several occurrences in this document of "video frame" or "audio sample" which could be shortened to "frame" or "sample" when it is clear by the dictionary name that we are talking about video or audio.

Reply:
I'll fix these in a new patch in https://github.com/w3c/webrtc-stats/pull/273 (which has these patches), as rebasing is painful.